# Stubs for networkx.tests.test_exceptions (Python 3)
#
# NOTE: This dynamically typed stub was automatically generated by stubgen.

def test_raises_networkxexception() -> None: ...
def test_raises_networkxerr() -> None: ...
def test_raises_networkx_pointless_concept() -> None: ...
def test_raises_networkxalgorithmerr() -> None: ...
def test_raises_networkx_unfeasible() -> None: ...
def test_raises_networkx_no_path() -> None: ...
def test_raises_networkx_unbounded() -> None: ...
